ConsoleApplication/Enum.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApplication
+{
+    enum TransactionType
```

Comment:
Please rename containing class to TransactionType.cs

Reply:
Thank you! Renamed